### PR TITLE
Display Notification Setting Status

### PR DESCRIPTION
**Pull request by warrickct**
* Adds an icon to conversation list items.
* Adds conversation screen subtitle showing the conversation's current notification setting.

truncated name disabled notifs

notification mentions only

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -416,5 +416,9 @@
   "pinConversationLimitToastDescription": "You can only pin $number$ conversations",
   "latestUnreadIsAbove": "First unread message is above",
   "sendRecoveryPhraseTitle": "Sending Recovery Phrase",
-  "sendRecoveryPhraseMessage": "You are attempting to send your recovery phrase which can be used to access your account. Are you sure you want to send this message?"
+  "sendRecoveryPhraseMessage": "You are attempting to send your recovery phrase which can be used to access your account. Are you sure you want to send this message?",
+  "all": "All",
+  "mentionsOnly": "Mentions only",
+  "disabled": "Disabled",
```
Comment (Bilb) [inline _locales/en/messages.json:422]: those three are already in notificationForConvo_mentions_only

File: ts/components/ConversationListItem.tsx [hunk 2]
```diff
@@ -95,7 +137,11 @@ const HeaderItem = (props: {
           profileName={profileName}
         />
       </div>
-      {pinIcon}
+
+      <StyledConversationListItemIconWrapper>
+        {pinIcon}
+        <NotificationSettingIcon />
```
Comment (Bilb) [inline ts/components/ConversationListItem.tsx:143]: you need to define a key for those two items
.you should be able to start the app with yarn start only to see this error

File: ts/components/conversation/ConversationHeader.tsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ const SelectionOverlay = (props: {
           iconType={SessionIconType.Exit}
           iconSize={SessionIconSize.Medium}
           onClick={onCloseOverlay}
-          theme={theme}
+          theme={useTheme()}
```
Comment (Bilb) [inline ts/components/conversation/ConversationHeader.tsx:89]: no need to pass the theme at all now that most of the content is under the Theme Provider

File: ts/components/conversation/ConversationHeader.tsx [hunk 4]
```diff
@@ -121,7 +120,7 @@ const TripleDotsMenu = (props: { triggerId: string; showBackButton: boolean }) =
       <SessionIconButton
         iconType={SessionIconType.Ellipses}
         iconSize={SessionIconSize.Medium}
-        theme={theme}
+        theme={useTheme()}
```
Comment (Bilb) [inline ts/components/conversation/ConversationHeader.tsx:123]: same

File: ts/components/conversation/ConversationHeader.tsx [hunk 5]
```diff
@@ -186,11 +184,29 @@ const BackButton = (props: { onGoBack: () => void; showBackButton: boolean }) =>
       iconSize={SessionIconSize.Large}
       iconRotation={90}
       onClick={onGoBack}
-      theme={theme}
+      theme={useTheme()}
```
Comment (Bilb) [inline ts/components/conversation/ConversationHeader.tsx:187]: same

File: ts/components/conversation/ConversationHeader.tsx [hunk 6]
```diff
@@ -245,21 +262,36 @@ const ConversationHeaderTitle = () => {
...
+  const notificationSubtitle = notificationSetting
+    ? window.i18n('notificationSubtitle', notificationSetting)
+    : null;
   const title = profileName || name || phoneNumber;
 
   return (
     <div className="module-conversation-header__title">
       <span className="module-contact-name__profile-name">{title}</span>
-      {textEl}
+      <StyledSubtitleContainer margin={useTheme().common.margins.xs}>
```
Comment (Bilb) [inline ts/components/conversation/ConversationHeader.tsx:274]: I'd like to have the theme extracted in a variable instead

File: ts/components/conversation/ConversationHeader.tsx [hunk 7]
```diff
@@ -245,21 +262,36 @@ const ConversationHeaderTitle = () => {
...
+        {isKickedFromGroup ? null : <ConversationHeaderSubtitle text={text} />}
+        <ConversationHeaderSubtitle text={notificationSubtitle} />
```
Comment (Bilb) [inline ts/components/conversation/ConversationHeader.tsx:276]: one of those line should be removed right?

**Review by Bilb — CHANGES_REQUESTED** [review]

**Review by Brice-W — APPROVED** [review]
Looks good to me

File: ts/components/conversation/ConversationHeader.tsx (hunk 7 above)
Comment (warrickct) [reply ts/components/conversation/ConversationHeader.tsx:276]: Top line is for the conditionally rendered member count subtitle and the other is the notification setting. I can rename the variable for the member count so it's clearer.

**Review by Bilb — APPROVED** [review]

**Pull request merged**